src/main/java/uk/gov/justice/hmpps/prison/api/model/TransactionHistoryDto.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import lombok.Builder;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.TransactionHistory;
+
+@Builder
+public class TransactionHistoryDto {
+
+    public static TransactionHistoryDto toDto(TransactionHistory transactionHistory) {
```

Comment:
We don't normally map like this. There is usually single mapping function that lives in a transform type class, either placed in the transform namespace or a function inside the service.   

As for data types it just an entity and set of api models containing swagger docs etc. See the ones in the following namespace: **uk.gov.justice.hmpps.prison.api.model;**

Reply:
it's a just place holder method, will follow existing examples